### ISSUE TITLE
find_object won't show my 3d camera image

### ISSUE THREAD

**Issue by rosnoob95** [introlab/find-object#59]
Hi,
I'm currently working on my bachelor thesis with an Intel Realsense Camera D435. I'm using the find_object package in order to detect items in boxes with the aim to improve bin-picking.
Sadly, while setting up all the necessary packages in my workspace, i encountered the following problem:

![bildschirmfoto vom 2018-07-02 11-00-14](https://user-images.githubusercontent.com/40698256/42158281-4b0bb96a-7df0-11e8-9296-ef0806003fd8.png)

![bildschirmfoto vom 2018-07-02 11-00-14](https://user-images.githubusercontent.com/40698256/42158561-16a84b38-7df1-11e8-951c-6fb0bc4443eb.png)

The camera should be calibrated correctly, at least these three topics shown in the find_object window are being published to when i call rostopic list (/camera/color/image_raw, /camera/aligned_depth_to_color/image_raw and /camera/color/camera_info)

I also followed all the steps in the tutorial for showing 3d images (editing the launchfile "find_object_3d.launch")

The camera is also working flawlessly (image is shown after calling rqt).

Can anybody give me a hint about what could be missing or going wrong ?
I would appreciate any kind of help, since I'm fairly new to ROS and my fellow students who are using this package told me that it should be working with my configurations!

Thanks in advance,
Lukas

**Comment by matlabbe** [comment]
Can you do 
$ rostopic hz /camera/color/image_raw
$ rostopic hz /camera/aligned_depth_to_color/image_raw
$ rostopic hz /camera/color/camera_info
to make sure everything is actually published?

**Comment by rosnoob95** [comment]
Hello there, thanks for your answer and help.
As I said, all these topics appeared when listing my rostopics, but then I launched only the camera without the find_object node, and that was when I realized that the topic "aligned_depth_to_color" was only being subscribed to by find_object and not being published to by the camera and therefore only appeared when running both nodes. Kind of a silly beginner-mistake, my apologies. I fixed the whole issue by installing the newest camera version compatible with the realsense ROS-wrapper 2.0.3.
